zerver/models/realms.py

```diff
@@ -335,7 +335,7 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
 
     DEFAULT_NOTIFICATION_STREAM_NAME = "general"
     INITIAL_PRIVATE_STREAM_NAME = "core team"
-    STREAM_EVENTS_NOTIFICATION_TOPIC_NAME = gettext_lazy("stream events")
+    STREAM_EVENTS_NOTIFICATION_TOPIC_NAME = gettext_lazy("channel events")
```

Comment:
Kept this as a separate commit in case we wanted to do something different here. Note that any change here will mean a new topic for these messages.

Reply:
Hmm, might be best to leave this change for a separate PR, just because there's a world where we decide it's worth trying to move all the messages from that topic to "channel events", especially given that we're linking to that topic from settings now?

Reply:
Sure. But just a quick question, since we store a message's topic as a string, would we be able to update it for existing messages with a translated version of this stream name?

Reply:
This commit is still in the updated backend changes.

Reply:
This commit is still in the updated backend changes.